policy/modules/system/init.te

```diff
@@ -78,10 +79,8 @@ mls_trusted_object(initctl_t)
 
 type initrc_t, init_script_domain_type, init_run_all_scripts_domain;
 type initrc_exec_t, init_script_file_type;
-domain_type(initrc_t)
-domain_entry_file(initrc_t, initrc_exec_t)
+init_domain(initrc_t, initrc_exec_t)
```

Comment:
There is a `init_domtrans_script(init_t)` later in the file that can be dropped.

Reply:
What about this part of `init_domtrans_script` ?
```
ifdef(`enable_mcs',`
    range_transition $1 initrc_exec_t:process s0;
')

ifdef(`enable_mls',`
    range_transition $1 initrc_exec_t:process s0 - mls_systemhigh;
')
```

Reply:
Replaced with calls to `init_ranged_domain()` in similar ifdefs.

Reply:
the `init_ranged_domain()` calls would be in init.te

Reply:
Do you want me to add a call to `init_ranged_domain` instead of `init_domtrans_script` since I don't see how the `range_transition` statements are applied otherwise.

Reply:
Like this ?

Reply:
Yes.